### PR TITLE
implements action set for iOS

### PR DESCRIPTION
**Pull request by nesalang**

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRActionSetRenderer.mm [hunk 1]
```diff
@@ -110,4 +112,14 @@ - (UIView *)renderButtons:(ACRView *)rootView
     return containingView;
 }
 
++ (ACRCardElementType)elemType {
+    return ACRActionSet;
+}
+
+- (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup rootView:(ACRView *)rootView inputs:(NSArray *)inputs baseCardElement:(ACOBaseCardElement *)acoElem hostConfig:(ACOHostConfig *)acoConfig {
+        ACOAdaptiveCard *card = [[ACOAdaptiveCard alloc] init];
+        [card setCard:[[rootView card] card]];
```
Comment (bviglietta) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRActionSetRenderer.mm]: Fix indentation

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRegistration.mm [hunk 2]
```diff
@@ -73,6 +73,7 @@ - (instancetype) init
              [ACRContainerRenderer getInstance],  [NSNumber numberWithInt:(int)[ACRContainerRenderer elemType]],
              [ACRColumnSetRenderer getInstance],  [NSNumber numberWithInt:(int)[ACRColumnSetRenderer elemType]],
              [ACRColumnRenderer getInstance],     [NSNumber numberWithInt:(int)[ACRColumnRenderer elemType]],
+             [ACRActionSetRenderer getInstance],   [NSNumber numberWithInt:(int)[ACRActionSetRenderer elemType]],
```
Comment (almedina-ms) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRegistration.mm]: >  [](start = 50, length = 1)

[NIT] remove one space

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRActionSetRenderer.mm (hunk 1 above)
Comment (nesalang) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRActionSetRenderer.mm]: thanks

---
In reply to: [270518627](https://github.com/Microsoft/AdaptiveCards/pull/2582#discussion_r270518627) [](ancestors = 270518627)

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRegistration.mm (hunk 2 above)
Comment (nesalang) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRegistration.mm]: thanks

---
In reply to: [270519273](https://github.com/Microsoft/AdaptiveCards/pull/2582#discussion_r270519273) [](ancestors = 270519273)

**Pull request merged**